gradle/libs.versions.toml

```diff
@@ -290,15 +291,6 @@ managed-micronaut-acme = { module = "io.micronaut.acme:micronaut-acme", version.
 managed-micronaut-cassandra = { module = "io.micronaut.cassandra:micronaut-cassandra", version.ref = "managed-micronaut-cassandra" }
 managed-micronaut-discovery = { module = "io.micronaut.discovery:micronaut-discovery-client", version.ref = "managed-micronaut-discovery" }
 managed-micronaut-elasticsearch = { module = "io.micronaut.elasticsearch:micronaut-elasticsearch", version.ref = "managed-micronaut-elasticsearch" }
-managed-micronaut-gcp = { module = "io.micronaut.gcp:micronaut-gcp-common", version.ref = "managed-micronaut-gcp" }
```

Comment:
:x: This is a breaking change for users who depend on the `libs.micronaut.cassandra` notation. Instead, it's probably a good idea to use `managed-micronaut-gcp = { module = "io.micronaut.gcp:micronaut-gcp-common", version = "" }`

Reply:
errr, I mixed 2 lines, it's `libs.micronaut.gcp.tracing` and friends.

Reply:
Cool, how do they relate to libs.micronaut.cassandra?

I can't see anything in the cassandra project 🤔

Reply:
What I mean is that if a user imports the Micronaut catalog and uses the following notation:

```
implementation mn.micronat.gcp.tracing
```

then your change will break that, because the catalog entry will be removed. To preserve the backwards compatibility, you need to keep the entry in the catalog, and simply replace the `version.ref` with an empy version:

```
managed-micronaut-gcp-tracing = { module = "io.micronaut.gcp:micronaut-gcp-tracing", version = "" }
```

The reference to cassandra is just an artifact of my brain not being fully awake :)

Reply:
If you agree, I'll fix this, then raise an issue to go back and fix up all the ones I deleted before in another PR?

Reply:
Np!

Reply:
ie: all these:https://github.com/micronaut-projects/micronaut-core/pulls?q=is%3Apr+BOM+is%3Aclosed+milestone%3A3.4.0

Reply:
@melix Made that change locally

```diff
+managed-micronaut-gcp = { module = "io.micronaut.gcp:micronaut-gcp-common", version = "" }
+managed-micronaut-gcp-tracing = { module = "io.micronaut.gcp:micronaut-gcp-tracing", version = "" }
+managed-micronaut-gcp-function = { module = "io.micronaut.gcp:micronaut-gcp-function", version = "" }
+managed-micronaut-gcp-function-http = { module = "io.micronaut.gcp:micronaut-gcp-function-http", version = "" }
+managed-micronaut-gcp-function-http-test = { module = "io.micronaut.gcp:micronaut-gcp-function-http-test", version = "" }
+managed-micronaut-gcp-http-client = { module = "io.micronaut.gcp:micronaut-gcp-http-client", version = "" }
+managed-micronaut-gcp-logging = { module = "io.micronaut.gcp:micronaut-gcp-logging", version = "" }
+managed-micronaut-gcp-pubsub = { module = "io.micronaut.gcp:micronaut-gcp-pubsub", version = "" }
+managed-micronaut-gcp-secret-manager = { module = "io.micronaut.gcp:micronaut-gcp-secret-manager", version = "" }
```

and I see this when publishing locally:

```
> Task :bom:generatePomFileForMavenPublication
[WARNING] Didn't find library io.micronaut.gcp:micronaut-gcp-function in BOM file
[WARNING] Didn't find library io.micronaut.gcp:micronaut-gcp-function-http in BOM file
[WARNING] Didn't find library io.micronaut.gcp:micronaut-gcp-common in BOM file
[WARNING] Didn't find library io.micronaut.gcp:micronaut-gcp-function-http-test in BOM file
[WARNING] Didn't find library io.micronaut.gcp:micronaut-gcp-pubsub in BOM file
[WARNING] Didn't find library io.micronaut.gcp:micronaut-gcp-secret-manager in BOM file
[WARNING] Didn't find library io.micronaut.gcp:micronaut-gcp-tracing in BOM file
[WARNING] Didn't find library io.micronaut.gcp:micronaut-gcp-http-client in BOM file
[WARNING] Didn't find library io.micronaut.gcp:micronaut-gcp-logging in BOM file
```

Is this benign and safe to ignore?

Reply:
Actually, I don't see any of them in the `micronaut-bom-3.4.0-20220303.141628-1.toml` file, so I guess it's not having the desired effect?

Reply:
mmm, that's annoying. It means the Micronaut BOM plugin requires an update to handle those cases.

Reply:
Does gradle not pull in the catalog for the gcp bom?